src/Select/index.stories.js

```diff
@@ -86,7 +87,7 @@ export const InAForm = () => {
           )
         </p>
       </div>
-      <Select label="Account" onChange={setInputValue}>
+      <Select id={"2"} label="Account" onChange={setInputValue}>
```

Comment:
nitpick - I'd prefer to use non-numeric ids in the examples. For this one, something like `id="account-field"`

Reply:
(encouraging people to name the things reduces the chances of two ids colliding again)